<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="5" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-31T20:20:50 IST" name="Default suite" finished-at="2024-12-31T20:20:53 IST" duration-ms="3150">
    <groups>
    </groups>
    <test started-at="2024-12-31T20:20:50 IST" name="Default test" finished-at="2024-12-31T20:20:53 IST" duration-ms="3150">
      <class name="com.hotelbooking.tests.BookingTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.hotelbooking.tests.BookingTest@436813f3]" started-at="2024-12-31T20:20:50 IST" name="setup" finished-at="2024-12-31T20:20:50 IST" duration-ms="366" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="createBookingId()[pri:1, instance:com.hotelbooking.tests.BookingTest@436813f3]" started-at="2024-12-31T20:20:50 IST" name="createBookingId" finished-at="2024-12-31T20:20:53 IST" duration-ms="2747" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createBookingId -->
        <test-method is-config="true" signature="validateAssertions()[pri:0, instance:com.hotelbooking.tests.BookingTest@436813f3]" started-at="2024-12-31T20:20:53 IST" name="validateAssertions" finished-at="2024-12-31T20:20:53 IST" duration-ms="1" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
BOOKINGDATES should be {"checkin":"2024-02-23","checkout":"2024-10-23"}, but it's not expected [{"checkin":"2024-02-23","checkout":"2024-10-23"}] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
BOOKINGDATES should be {"checkin":"2024-02-23","checkout":"2024-10-23"}, but it's not expected [{"checkin":"2024-02-23","checkout":"2024-10-23"}] but found [null]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at base.BaseTest.validateAssertions(BaseTest.java:143)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:700)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:676)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:660)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateAssertions -->
      </class> <!-- com.hotelbooking.tests.BookingTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
